src/utils/__tests__/util.spec.ts

```diff
@@ -0,0 +1,40 @@
+import utils from '../util';
+
+describe('ext util', () => {
```

Comment:
Maybe move ext util functions to the ext folder?

Reply:
That was a leftover from before I refactored/cleaned the code